test/evergreen.js

```diff
@@ -30,9 +74,113 @@ test('findPlatforms: none', (t) => {
 });
 
 test('findPlatforms: android, ios', (t) => {
+  const evergreen = require(TEST_SUBJECT);
+
   return evergreen.findPlatforms(path.join(__dirname, 'fixtures', 'evergreen'))
     .then((platforms) => {
       t.is(Array.isArray(platforms), true);
       t.deepEqual(platforms, ['android', 'ios']);
     });
 });
+
+test('assumeAWSRole() should resolve', (t) => {
+  const evergreen = t.context.getTestSubject();
+
+  return evergreen.assumeAWSRole();
+});
+
+test('getCurrentTenant() should resolve to current tenant', (t) => {
+  const evergreen = t.context.getTestSubject();
+
+  return evergreen.getCurrentTenant()
+    .then((tenant) => t.is(tenant, TENANT_NAME));
+});
+
+test('getCurrentTenant() should reject if no current tenant is found', (t) => {
+  const evergreen = t.context.getTestSubject({
+    '@blinkmobile/bm-identity': function BlinkMobileIdentity () {
+      this.getTenants = () => {
+        return Promise.resolve();
+      };
```

Comment:
Shorter (although not necessarily clearer): `this.getTenants = () => Promise.resolve()`?